code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/energy/e_gun
 	name = "energy gun"
-	desc = "A basic hybrid energy gun with two settings: disable and kill."
+	desc = "A basic energy gun, with two settings: disable and kill."
```

Comment:
It is in fact a hybrid energy gun, full size laser guns only have the kill lasdrs